src/React.Core/IReactEnvironment.cs

```diff
@@ -110,5 +109,10 @@ public interface IReactEnvironment
 		/// Returns the currently held JS engine to the pool. (no-op if engine pooling is disabled)
 		/// </summary>
 		void ReturnEngineToPool();
+
+		/// <summary>
+		/// Gets the site-wide configuration.
+		/// </summary>
+		IReactSiteConfiguration Configuration { get; }
```

Comment:
Isn't there already a getter for this? `ReactSiteConfiguration.Configuration` or `.Current` or something like that?